data/json/monsters/fungus_zombie.json

```diff
@@ -0,0 +1,259 @@
+      "armor_cut": 22,
+      "armor_bullet": 30,
+      "armor_stab": 30,
+      "armor_acid": 3
+    },
+    "special_attacks": [ [ "FUNGUS", 200 ] ],
+    "extend": { "flags": [ "HARDTOSHOOT" ] },
+    "harvest": "mr_bones",
+    "death_function": { "corpse_type": "NO_CORPSE", "effect": { "id": "death_fungus", "hit_self": true } }
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_skeleton_brute_fungus",
+    "name": { "str": "fungal skeletal brute" },
+    "copy-from": "mon_skeleton_fungus",
+    "looks_like": "mon_skeleton_brute",
+    "color": "white",
+    "material": [ "bone" ],
+    "description": "Fungi bloom from the crevices in the twisted bone growths that cover this creature.  Spores linger in the air around it.",
+    "proportional": { "hp": 1.5, "speed": 1.5 },
+    "relative": { "melee_dice": 1, "melee_dice_sides": 5, "melee_cut": 2, "armor_bash": 4, "armor_cut": 6, "armor_bullet": 5 },
+    "special_attacks": [ [ "SMASH", 30 ], [ "FUNGUS", 200 ] ],
+    "extend": { "flags": [ "GROUP_BASH", "PUSH_MON", "PUSH_VEH", "HARDTOSHOOT" ] },
+    "harvest": "mr_bones",
+    "death_function": { "corpse_type": "NO_CORPSE", "effect": { "id": "death_fungus", "hit_self": true } }
+  },
+  {
+    "id": "mon_skeleton_hulk_fungus",
+    "type": "MONSTER",
+    "name": { "str": "fungal juggernaut" },
+    "looks_like": "mon_skeleton_hulk",
+    "description": "Fungi bloom from the crevices in the ossified plates of this ponderous bone titan, and it seems even its eyes are lost to it.  A dust of spores lands on the ground with every sluggish stomp of its heavy legs.",
+    "default_faction": "fungus",
+    "bodytype": "human",
+    "species": [ "FUNGUS" ],
+    "diff": 2,
+    "volume": "875000 ml",
+    "weight": "200 kg",
+    "hp": 300,
+    "speed": 30,
+    "material": [ "bone" ],
+    "symbol": "Z",
+    "color": "white",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_cut": 15,
+    "armor_bash": 20,
+    "armor_cut": 45,
+    "armor_bullet": 36,
+    "bleed_rate": 0,
+    "vision_day": 5,
+    "vision_night": 5,
+    "harvest": "mr_bones",
+    "special_attacks": [ [ "SMASH", 20 ], [ "FUNGUS", 200 ] ],
+    "death_drops": "mon_zombie_hulk_death_drops",
+    "death_function": { "corpse_type": "NO_CORPSE", "effect": { "id": "death_fungus", "hit_self": true } },
+    "burn_into": "mon_zombie_fiend",
+    "flags": [
+      "HEARS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "DESTROYS",
+      "POISON",
+      "NO_BREATHE",
+      "PUSH_MON",
+      "PUSH_VEH",
+      "HARDTOSHOOT",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_child_fungus",
+    "type": "MONSTER",
+    "name": { "str": "fungal child", "str_pl": "fungal children" },
+    "description": "It is hard to recognize a human child in this creature.  Disgusting-looking mold covers most of its skin.  Many cracks perforate its body, with small fungal stalks poking through.",
```

Comment:
Should this both be described as "hard to recognize a human child" and have GUILT? Possibly not a matter for this PR, however.

Reply:
I'm already in the neighborhood. Agreed.